frontend/src/pages/event.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react'
+import { Table, Row, Button, Col, FormGroup, Label, Input, Container } from 'reactstrap'
+import ActionButton from '../components/actionButton'
+import Nav from '../components/nav'
+import Head from '../components/head'
+import { getEventById, getEventAttendees } from '../utils/api'
+import Router from 'next/router'
+import Link from 'next/link'
+
+class Event extends React.Component<Props> {
```

Comment:
Tiny thing but could just say `Component` here since it's imported, or otherwise remove `Component` from the imports

Reply:
Based on @n3a9 's refactor I'd say just extend `Component`